apps/internal-tool/.env.development

```
@@ -1,6 +1,19 @@
-NEXT_PUBLIC_STACK_PROJECT_ID=internal
-NEXT_PUBLIC_STACK_PUBLISHABLE_CLIENT_KEY=this-publishable-client-key-is-for-local-development-only
-NEXT_PUBLIC_STACK_DASHBOARD_URL=http://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}01
+NEXT_PUBLIC_HEXCLAVE_API_URL=http://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}02
+NEXT_PUBLIC_HEXCLAVE_PROJECT_ID=internal
+NEXT_PUBLIC_HEXCLAVE_PUBLISHABLE_CLIENT_KEY=this-publishable-client-key-is-for-local-development-only
+NEXT_PUBLIC_HEXCLAVE_DASHBOARD_URL=http://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}01
 NEXT_PUBLIC_SPACETIMEDB_HOST=ws://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}39
-NEXT_PUBLIC_SPACETIMEDB_DB_NAME=stack-auth-llm
+NEXT_PUBLIC_SPACETIMEDB_DB_NAME=hexclave-ai-analytics
+# Server-side SpacetimeDB access (this app owns all reducer/SQL calls)
+STACK_SPACETIMEDB_URL=http://localhost:${NEXT_PUBLIC_HEXCLAVE_PORT_PREFIX:-81}39
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The STACK_SPACETIMEDB_URL key is not assigned properly

(SubstitutionKey)